### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131201%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#11] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131201%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

聚會時間/地點

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1131201%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
『那時，諸天的國好比十個童女，拿著她們的燈，出去迎接新郎。』太二十五1

『其中五個是愚拙的，五個是精明的。』太二十五2
『愚拙的拿著她們的燈，卻沒有帶著油；』太二十五3

『但精明的拿著她們的燈，又在器皿裏帶著油。』太二十五4

『新郎遲延的時候，她們都打盹睡著了。』太二十五5

『半夜有人喊叫：看哪，新郎來了，你們出來迎接他。』太二十五6

『那些童女就都起來，整理她們的燈。』太二十五7

『愚拙的對精明的說，請分點油給我們，因為我們的燈要滅了。』太二十五8

『精明的回答說，恐怕不彀我們和你們用的，不如你們到賣油的那裏，為自己買罷。』太二十五9

『不料，她們去買的時候，新郎到了；那豫備好的，同他進去赴婚筵，門就關了。』太二十五10

『其餘的童女隨後也來了，說，主阿，主阿，給我們開門。』太二十五11

『他卻回答說，我實在告訴你們，我不認識你們。』太二十五12

『所以你們要儆醒，因為那日子、那時辰，你們不知道。』太二十五13

『但是你們可以從無花果樹學個比方：當樹枝發嫩長葉的時候，你們就知道夏天近了；』太二十四32

『照樣，你們幾時看見這一切的事，也該知道那夏天近了，正在門口了。』太二十四33

『我實在告訴你們，這一代還沒有過去，這一切事都要發生。』太二十四34
